### DISCUSSION TITLE
WSDD works on wired but not on wireless connection on the same network

### DISCUSSION THREAD

**Discussion by antolatella** [christgau/wsdd#115] [Q&A] [answered]
I have this network configuration:
- My home network, with a Router (wireless/wired connection),
- A Linux Machine. It runs Samba and wsdd. It's wired-connected to Router,
- A Windows Machine.

By connecting the windows machine to the router via cable, it manages to discover the linux device, which is displayed with its choosen network name.

Instead, by connecting the windows machine to the router via wireless, the discovery does not work (the linux device is not displayed in the list of network devices), but by writing the IP address of the device in the explorer address bar, I can still reach it ( the network name of the linux device is not displayed, only its ip address).
Could this be due to a network configuration or WSDD problem?

Thank you

**Comment by christgau** [comment, marked as answer]
I have a similar setup here that was used during most if not all stages of development. I primarily use a Windows host that has a cabled connection and a wifi-connected host for developing wsdd. However, I also use wifi-connected Windows hosts and wsdd hosts that are connected via cable. There isn't any problem for all Windows hosts do discover the any of the  wsdd hosts. All of hosts share the same IP subnet which is an essential thing here. Please check that is the case for your setup as well. You could also check that wsdd binds to the appropriate interface and that your firewall allows the wsdd-related traffic.

I realize that you are able to connect to the Samba host, so TCP appears to be working here, but please note that connecting to a Samba host has nothing to do with wsdd itself.
